src/components/DeckBuilderDryRunView/index.js

```diff
@@ -54,7 +56,14 @@ export default props => {
   const deck = modifyDeck(props.deck, modifier, equalsMode).map(addIdx)
 
   return (
-    <DeckMechanisms deck={deck} mode={mode} equalsMode={equalsMode}>
+    <DeckMechanisms
+      deck={deck}
+      mode={mode}
+      equalsMode={equalsMode}
+      harvestersCards={harvestersCards}
+      setHarvestersCards={setHarvestersCards}
+      harvestersDialogRef={harvestersDialogRef}
```

Comment:
Maybe we should pass a `HoS` prop? Which means we could just pass that object to `handleCardEffect`.

```jsx
HoS={{ cards, setCards, dialog }}
```